.github/workflows/tool-release-docker.yaml

```diff
@@ -200,7 +200,7 @@ jobs:
           docker push ${{ secrets.CONTAINER_REGISTRY_URL }}/${{ matrix.docker.team_name }}/${{ matrix.docker.image_name }}:${{ inputs.docker_tag_override }}
 
       - name: Push docker image with latest tag
-        if: inputs.docker_use_latest == 'true' && inputs.docker_push == 'true' && env.IS_PRE_RELEASE == 'false' && env.DOCKER_DISABLED != 'true'
+        if: matrix.docker.use_latest == true && inputs.push_docker  == 'true' && env.DOCKER_DISABLED != 'true'
```

Comment:
seems like there's an unintentional extra whitespace  between `inputs.push_docker` and `==`